### PR TITLE
refactor: add lazy getters

### PR DESCRIPTION
**Pull request by caspiano**
- Adds lazy getters to controllers
- More unique names for testing updates
- Fix a boolean parameter bug in modules controller

### CODE REVIEW

**Review by stakach — APPROVED** [review]
nice! LGTM

**Review by GabFitzgerald — APPROVED** [review]

File: src/placeos-rest-api/controllers/modules.cr [hunk 1]
```diff
@@ -65,14 +65,14 @@ module PlaceOS::Api
           query.filter({"driver_id" => [driver_id]})
         end
 
-        if (connected = args.connected)
+        unless (connected = args.connected).nil?
```
Comment (dukeraphaelng) [inline src/placeos-rest-api/controllers/modules.cr:68]: I have never seen this pattern before. Why is assignment put as a condition in an if/else cause? Does it return a boolean value?

Comment (caspiano) [reply src/placeos-rest-api/controllers/modules.cr:68]: It's a tight (albeit indirect) way of doing an assignment.
Assignment expressions in crystal return the value of the assignment, so you get away with what's happening here.

**Review by dukeraphaelng — APPROVED** [review]
Everything is good, I just have one question on line 68 in src/placeos-rest-api/controllers/modules.cr

**Pull request merged**